src/main/webapp/app/course/course-scores/course-scores-csv-row-builder.ts

```diff
@@ -45,21 +41,35 @@ export class CourseScoresCsvRowBuilder {
     }
 
     /**
-     * Stores the given value under the key in the row after converting it to the localized format.
+     * Stores the given value under the key in the row after converting it to the format using the specified decimal separator.
      * @param key Which should be associated with the given value.
      * @param value That should be placed in the row.
      */
     setLocalized(key: string, value: number) {
-        this.set(key, this.localizer(value));
+        const options: Intl.NumberFormatOptions = {
```

Comment:
Thanks.

The formatting of the number now again depends on the browser settings of the user. I can see two ways in which this might break:
- language set to German, but output should be with `.`: E.g., for `23,5` the replacement won’t change that.
- `toLocaleString` seems to [insert thousands-separators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_tolocalestring) as well. Therefore, the replacement would change e.g., `1.000,23` to `1,000,23`.

Maybe it would be best to use the [existing rounding utility](https://github.com/ls1intum/Artemis/blob/6ec8c5c6f257c7ef7229e379650148f0522d9792/src/main/webapp/app/shared/util/utils.ts#L100) and then use the regular `toString` on the resulting number to avoid those two issues.